pages/index.tsx

```diff
@@ -22,9 +22,12 @@ import {
 	timeCamp,
 } from "@/public/assets/images/landing-page";
 import Image from "next/image";
+import { useRouter } from "next/router";
 import { useState } from "react";
 
 const HomePage = () => {
+	const router = useRouter();
+
 	const trustedPioneers = [
```

Comment:
Please move all static data like objects and arrays and such outside of the component... As long as they don't depend on anything within the component, please do well to move them out